manifests/m/Mozilla/Firefox/100.0/Mozilla.Firefox.installer.yaml

```
@@ -16,783 +16,7553 @@ FileExtensions:
-    DisplayVersion: "100.0"
-    Publisher: Mozilla
-    ProductCode: Mozilla Firefox 100.0 (x86 nb-NO)
-- InstallerLocale: nb-NO
-  Architecture: x86
-  InstallerType: exe
-  Scope: machine
-  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/100.0/win32/nb-NO/Firefox%20Setup%20100.0.exe
-  InstallerSha256: 12AAEFB4E2318BECC6BFBC0A2D54BAA519FBFE7557575FFC1FBBADD1201DFA45
-  InstallerSwitches:
-    Silent: /S
-    SilentWithProgress: /S
-  AppsAndFeaturesEntries:
-  - DisplayName: Mozilla Maintenance Service
-    DisplayVersion: "100.0"
-    Publisher: Mozilla
-    ProductCode: MozillaMaintenanceService
-  - DisplayName: Mozilla Firefox (x86 nb-NO)
-    DisplayVersion: "100.0"
-    Publisher: Mozilla
-    ProductCode: Mozilla Firefox 100.0 (x86 nb-NO)
-- InstallerLocale: nb-NO
-  Architecture: arm64
-  InstallerType: exe
-  Scope: machine
-  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/100.0/win64-aarch64/nb-NO/Firefox%20Setup%20100.0.exe
-  InstallerSha256: 119885819B1B73AEA045695D3920916378995A5879EC52EE99F9DCF259DE9339
-  InstallerSwitches:
-    Silent: /S
-    SilentWithProgress: /S
-- Architecture: x64
-  InstallerType: msix
-  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/100.0/win64/multi/Firefox%20Setup%20100.0.msix
-  InstallerSha256: 531EB3AF67BA44C8C9A8423FA25A9D9EC530F7F9FC3A6AA3C70826580987A600
-  SignatureSha256: 8559425028c96b74c3b037433ad171c8d3dc8b7d119fdfadfce790db973b0129
-  PackageFamilyName: Mozilla.MozillaFirefox_gmpnhwe7bv608
-- Architecture: x86
-  InstallerType: msix
-  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/100.0/win32/multi/Firefox%20Setup%20100.0.msix
-  InstallerSha256: D2DAC65EB7C2701B79965022B01EE8693CB1EC2CB87A12A5E1DACAF272C625BC
-  SignatureSha256: 756cb29bdcdb1641f6b08b65d68d5c575024d21ae0db55a96e0186c2cf0b160b
-  PackageFamilyName: Mozilla.MozillaFirefox_gmpnhwe7bv608
+  - InstallerLocale: ach
+    Architecture: x64
+    InstallerType: exe
+    Scope: machine
+    InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/100.0/win64/ach/Firefox%20Setup%20100.0.exe
+    InstallerSha256: 5EC7A492F2A3DCC829DF32C7B3ECE6C5E7519F9DCF979D6642CEEF2D16750120
+    InstallerSwitches:
+      SilentWithProgress: /S
+      Silent: /S
+    AppsAndFeaturesEntries:
+    - Publisher: Mozilla
+      DisplayVersion: "100.0"
+      DisplayName: Mozilla Firefox (x64 ach)
+      ProductCode: Mozilla Firefox 100.0 (x64 ach)
+    - Publisher: Mozilla
+      DisplayVersion: "100.0"
+      DisplayName: Mozilla Maintenance Service
+      ProductCode: MozillaMaintenanceService
+  - InstallerLocale: ach
+    Architecture: x64
+    InstallerType: wix
+    Scope: machine
+    InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/100.0/win64/ach/Firefox%20Setup%20100.0.msi
+    InstallerSha256: 088C6A1CAC8A92303679C298CEC45188BC9A11277DEA8AC9DE0C0E427D9E2B98
```

```suggestion
    InstallerSha256: 088C6A1CAC8A92303679C298CEC45188BC9A11277DEA8AC9DE0C0E427D9E2B98
    InstallerSwitches:
      InstallLocation: INSTALL_DIRECTORY_PATH="<INSTALLPATH>"
```
https://searchfox.org/mozilla-central/source/browser/installer/windows/msi/installer.wxs